ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/response/CCDFullAdmissionResponse.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.ccd.domain.response;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDParty;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCDRepaymentPlan;
+import uk.gov.hmcts.cmc.ccd.domain.CCDStatementOfTruth;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDStatementOfMeans;
+
+import java.time.LocalDate;
+
+@Builder
+@Value
+public class CCDFullAdmissionResponse {
```

Comment:
Please move the following to `CCDAbstractResponse`:

```
private CCDParty defendant;
private CCDYesNoOption freeMediationOption;
private CCDStatementOfTruth statementOfTruth;
```

Then make both `CCDFullAdmissionResponse` and `CCDFullDefenceResponse` extend abstract class.

Reply:
Not implementing after discussion as `@Builder` in current release of lombok does not support for members from ancestors. We will wait for the `@SuperBuilder` coming in upcoming official release and will clean up this tech debt later.